core/core-impl/src/main/java/ai/timefold/solver/core/api/solver/SolverManager.java

```diff
@@ -98,6 +99,17 @@ static <Solution_, ProblemId_> SolverManager<Solution_, ProblemId_> create(
         return new DefaultSolverManager<>(solverFactory, solverManagerConfig);
     }
 
+    // ************************************************************************
+    // Builder method
+    // ************************************************************************
+
+    /**
+     * Creates a Builder that allows to customize and submit a planning problem to solve.
+     *
+     * @return never null
+     */
+    SolverJobBuilder<Solution_, ProblemId_> buildSolver();
```

Comment:
I think we agreed that this method would be called `solveBuilder()`. To be consistent with the `solve()` methods.